simapp/go.mod

```diff
@@ -154,7 +154,6 @@ require (
 
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
-	github.com/cosmos/cosmos-sdk => ../.
```

Comment:
We opted to leave this instead of updating the import each time. Can we still leave it?

Reply:
Not if we want external apps to be able to use simapp

Reply:
*maybe now that the version is fixed? 

I'll check this out locally and let you know

Reply:
which external apps are using simapp? generally I don think anyone should be using simapp.

Reply:
Interesting. Wasmd.

Reply:
I agree with Marko, the goal is to not have apps import simapp. If this is still required, this means we have forgotten something. Looking at wasmd, it looks like we forgot to extract the App interface. PR incoming.